composer.json

```diff
@@ -59,6 +59,7 @@
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
         "oat-sa/lib-lti1p3-ags": "^1.2",
         "oat-sa/lib-lti1p3-core": "^6.0.0",
+        "oat-sa/extension-tao-delivery" : ">=15.6.0",
```

Comment:
This is technically a breaking change, although, obviously that extension would be present in any case.

Let's highlight the dependency in the release notes anyway, and bump the major version.
You can do that by adding an exclamation mark to your commit message prefix, e.g. ```chore!: introduce an `oat-sa/extension-tao-delivery` dependency```.